editor/src/components/code-editor/script-editor.tsx

```diff
@@ -348,8 +348,7 @@ export const ScriptEditor = betterReactMemo('ScriptEditor', (props: ScriptEditor
   const newErrorsForFile = errors.filter((error) => error.fileName === filePath)
   const errorsForFile = useKeepReferenceEqualityIfPossible(newErrorsForFile)
 
-  const newNpmDependencies = dependenciesFromPackageJson(packageJsonFile)
-  const npmDependencies = useKeepReferenceEqualityIfPossible(newNpmDependencies)
+  const npmDependencies = usePossiblyResolvedPackageDependencies()
```

Comment:
was this `useKeepReferenceEqualityIfPossible` redundant here? why was this added in the first place? was there a reason or was it just unnecessary caution?

Reply:
Not sure about the original case, but this _should_ be fine as there's some `useMemo` inside it for that kind of reason.